scripts/core-ng-module-generator.js

```diff
@@ -133,6 +127,8 @@ function createDirective(element, module) {
 function getDirectiveProps(props) {
   return props
     .filter(prop => prop.privacy === undefined) // public
+    .filter(prop => !prop.static)
+    .filter(prop => !prop.readonly)
```

Comment:
Improve generated types by excluding readonly/static props from being generated as `@input`s on the directives